client/my-sites/email/email-providers-stacked-comparison/index.tsx

```diff
@@ -171,16 +175,17 @@ const EmailProvidersStackedComparison = ( {
 				onIntervalChange={ changeIntervalLength }
 			/>
 
-			{ hasExistingEmailForwards && domainsWithForwards !== undefined && (
+			{ ! hasCartDomain && hasExistingEmailForwards && domainsWithForwards !== undefined && (
 				<EmailExistingForwardsNotice
 					domainsWithForwards={ domainsWithForwards }
 					selectedDomainName={ selectedDomainName }
 				/>
 			) }
 
-			<EmailExistingPaidServiceNotice domain={ domain } />
+			{ ! hasCartDomain && domain && <EmailExistingPaidServiceNotice domain={ domain } /> }
 
 			<ProfessionalEmailCard
+				cartDomainName={ cartDomainName }
```

Comment:
Similarly to another comment, I would rather pass a boolean than having a new prop with the same domain name than `selectedDomainName`.